components/gitpod-protocol/src/public-api-utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { PaginationRequest } from "@gitpod/public-api/lib/gitpod/v1/pagination_pb";
```

Comment:
I don't match like utils in file names but it is nit. I would rather just call public-api-pagination.ts

I noticed that we were doing the same in several PRs. cc @AlexTugarev @selfcontained 

It would be good to align how we do it and there to put. We could put in public-api typescript that each client gets helper functions, then we call this file pagination.ts

Reply:
It's called utils because I added orderBy related function here before XD

Reply:
not a blocker to me land